collect_app/src/main/res/values/strings.xml

```diff
@@ -338,11 +338,11 @@
     <string name="get_trace">Start GeoTrace</string>
 
     <string name="geoshape_title">GeoShape</string>
-    <string name="select_geotrace_mode">"Select GeoTrace Mode"</string>
+    <string name="data_entry_mode">Data entry mode</string>
```

Comment:
Data entry is an overloaded term. Maybe "Placement mode"

Reply:
I'll address this together with the other comment below.